File: web-app/backend/post.js
var request = require('request');
var options = {
  uri: "http://fluentd:9999/sample.data",
  headers: {
    "Content-type": "application/json",
  },
  json: {
    "key1": "param1",
    "key2": "param2"
  }
};
request.post(options, function(error, response, body){
  console.log(response.statusCode);
});

File: python/post.py
import requests
import time

# 5秒に1回POSTリクエストを送信する
url = 'http://fluentd:9999/sample.data'
i = 0
while True:
    payload = {"id":i}
    requests.post(url, json=payload)
    print('id:', i)
    i += 1
    time.sleep(1)